plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -1840,7 +1840,7 @@ instance uni ~ DefaultUni => ToBuiltinMeaning uni DefaultFun where
     -}
 
 instance Default (BuiltinSemanticsVariant DefaultFun) where
-    def = DefaultFunSemanticsVariant2
+    def = DefaultFunSemanticsVariantC
```

Comment:
Could've just probably made it `def = maxBound` BTW.

Reply:
Y I start liking Bounded more than Default

Reply:
> Y I start liking Bounded more than Default

Good idea.  There are a lot of `def`s in various places and it can be quite hard to tell exactly what the default is.